lib/presentation/views/notification/notification_info.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class NotificationInfoView extends StatelessWidget {
+  final String message;
```

Comment:
Please move this line after the constructor to keep it consistent, it's more readable that way

Reply:
not sure what line you're refering to remove sorry, as in the whole file?

Reply:
done